modules/flight_interface/flight_interface.py

```diff
@@ -9,6 +9,7 @@
 from ..logger import logger
 from ..common.mavlink.modules import drone_odometry
 from ..common.mavlink.modules import flight_controller
+from ..decision_command import DecisionCommand
```

Comment:
to follow the design pattern, use from .. import decision_command instead (to match the importing of odometry_and_time